src/player-data.ts

```diff
@@ -215,21 +215,21 @@ export default class PlayerData extends EventEmitter {
     }
   }
 
-  chargeGaiaPower(power: number){
-    if ( this.brainstone === BrainstoneArea.Transit){
+  chargeGaiaPower(power: number) {
+    if ( this.brainstone === BrainstoneArea.Transit) {
       this.brainstone = BrainstoneArea.Gaia;
       power -= 1;
     }
     this.power.gaia += power;
-  };
+  }
 
-  discardGaiaPower(power: number){
-    if ( this.brainstone === BrainstoneArea.Gaia){
-      this.brainstone = BrainstoneArea.Transit;
-      power -= 1;
-    }
+  returnGaiaPower(power: number) {
+    // this is used only by Terrans for the conversion
+    // they are pay tg to get x
+    this.power.area1 += this.hasPlanetaryInstitute() ? 0 : power;
```

Comment:
This function is only called if Terrans have their planetary institute. Terrans always get their power tokens in area 2, even if they have no planetary institute.

Reply:
oh right, sorry